src/yargs-modules/shell.ts

```diff
@@ -0,0 +1,44 @@
+import mkCmd from "../utils/cmd-handler";
+import { Argv } from "yargs";
+
+const Shell = (module.exports = {
```

Comment:
More mixed module systems. I've found that this can cause issues for folks who aren't super familiar with the codebase, especially as projects grow.

Reply:
Please see https://github.com/entropic-dev/ds/pull/2#discussion_r301457809